docs/en/token_endpoint.rst

```diff
@@ -301,6 +301,60 @@ The claims available in the *ID Token* are given below.
  - https://openid.net/specs/openid-connect-core-1_0.html#IDToken
  - https://openid.net/specs/openid-igov-openid-connect-1_0-03.html#Section-3.1
 
+Refresh Token
++++++++++++++
+
+The *Refresh Token* is a signed JWT that MAY be issued by the OP and MAY be used to obtain a new *Access Token* that enables the RP to access the UserInfo endpoint without direct user interaction. To obtain a *Refresh Token*, the RP MUST include in the scope parameter of the authentication request the value offline_access. The use of this scope can be useful in scenarios where an RP needs to verify that an end user's digital identity is still valid (e.g., it has not been revoked) or wants to keep attributes it has previously collected during the authentication phase up-to-date, such as for sending notifications to the end user after the end user has been authenticated.
+
+The *Refresh Token* MUST contain at least the following parameters.
+
+.. list-table:: 
+  :widths: 20 60 20
+  :header-rows: 1
+
+  * - **Claim**
+    - **Description**
+    - **Supported by**
+  * - **iss** 
+    - It MUST be an HTTPS URL that uniquely identifies the OP. The RP MUST verify that this value matches the called OP.
+    - |spid-icon| |cieid-icon|
+  * - **aud** 
+    - It MUST match the value client_id. The RP MUST verify that this value matches its client ID.
+    - |spid-icon| |cieid-icon|
+  * - **iat** 
+    - UNIX Timestamp with the time of JWT generation, coded as NumericDate as indicated in :rfc:`7519`.
+    - |spid-icon| |cieid-icon|
+  * - **exp** 
+    - UNIX Timestamp with the expiry time of the JWT, coded as NumericDate as indicated in :rfc:`7519`.
+    - |spid-icon| |cieid-icon|
+  * - **jti** 
+    - It MUST be a String in uuid4 format. Unique *Refresh Token* identifier that the RP MAY use to prevent reuse by rejecting the *Refresh Token*  if already processed.
+    - |spid-icon| |cieid-icon|
+
+.. warning::
+  
+  **The Refresh Token MUST NOT allow the requesting RP to obtain an ID Token, neither the one previously issued during authentication nor a new ID Token. The use of the Refresh Token MUST NOT be used by RPs to obtain a new user authentication with the OP, but MAY be used as a mechanism to obtain from the UserInfo endpoint only the same set of user attributes requested at the initial authentication phase and for which the user has given explicit consent.** Such consent MUST be collected by the OP at the end-user authentication phase on the consent page (it requires the ability to enable or disable this option on the consent page) and MAY be conditional on a validity period if defined by the OP according to policies on the treatment of personal data. Disabling this option by the user MUST still allow user authentication but MUST NOT release a *Refresh Token*. 
```

Comment:
Ok with this

a subissue should be openend to have the same UI/UX to all the OP for this feature